.github/workflows/code-checker.yml

```
@@ -95,7 +95,7 @@ jobs:
     name: Semgrep
     runs-on: ubuntu-latest
     container:
-      image: returntocorp/semgrep@sha256:ffc6f3567654f9431456d49fd059dfe548f007c494a7eb6cd5a1a3e50d813fb3
+      image: returntocorp/semgrep:1.58
```

```suggestion
      image: returntocorp/semgrep@sha256:cfad18cfb6536aa48ad5a71017207a10320b4e17e3b2bd7b7de27b42dc9651e7 #v1.58
```

I am not sure if we have to use digests for containers the way we do for reusable GH Actions, but here is the SHA form if we want to stay consistent with what it looked like before.

---

Thanks!  Yeah, looks like based on the TSCCR that's what we should be doing.